src/hdtSkinnedMesh/hdtCollider.cpp

```
@@ -110,24 +150,54 @@ namespace hdt
+			else {
+				node->aabbMe.invalidate();
+			}
 #endif
 
-		aabbAll = aabbMe;
-		for (auto& i : children) {
-			// FIXME PROFILING Lots of time is used here, because this is called a lot.
-			i.updateAabb();
-
-			aabbAll.merge(i.aabbAll);
+			__m128 allMin = node->aabbMe.m_min;
```

Clang (14.0.6)

[warning] variable 'allMin' is not initialized

(cppcoreguidelines-init-variables)